### PR TITLE
Apply black style to Bio.phenotype.

### PR DESCRIPTION
**Pull request by svalqui**
This pull request addresses issue #2008 

- [X] I hereby agree to dual licence this and any previous contributions under both
the _Biopython License Agreement_ **AND** the _BSD 3-Clause License_.

- [X] I have read the ``CONTRIBUTING.rst`` file, have run ``flake8`` locally, and
understand that AppVeyor and TravisCI will be used to confirm the Biopython unit
tests and style checks pass with these changes.

- [X] I have added my name to the alphabetical contributors listings in the files
``NEWS.rst`` and ``CONTRIB.rst`` as part of this pull request, am listed
already, or do not wish to be listed. (*This acknowledgement is optional.*)

Should be fine to merge.

### CODE REVIEW

File: Bio/phenotype/phen_micro.py [hunk 1]
```diff
@@ -182,8 +183,9 @@ def _is_well(self, obj):
         """
         # Value should be of WellRecord type
         if not isinstance(obj, WellRecord):
-            raise ValueError("A WellRecord type object is needed as value" +
-                             " (got %s)" % type(obj))
+            raise ValueError(
+                "A WellRecord type object is needed as value" + " (got %s)" % type(obj)
```
Comment (peterjc) [inline Bio/phenotype/phen_micro.py]: We can remove the plus and combine these strings.

(The original code had a redundant plus)

File: Bio/phenotype/pm_fitting.py [hunk 2]
```diff
@@ -54,8 +54,17 @@ def richards(x, A, u, d, v, y0):
...
+        A
+        * pow(
+            1
+            + (
+                v
+                + (np.exp(1 + v) * np.exp((u / A) * (1 + v) * (1 + (1 / v)) * (d - x)))
+            ),
+            -(1 / v),
+        )
+    ) + y0
```
Comment (peterjc) [inline Bio/phenotype/pm_fitting.py]: Well that is a long looking expression - but the original was already rather long. The single letter variable names are also less than ideal, but neither is black's fault.

**Comment by peterjc** [comment]
One string addition to fix, and this looks ready to merge. Thanks.

**Comment by svalqui** [comment]
@peterjc Thank you for the review/feedback amended.

File: Bio/phenotype/pm_fitting.py (hunk 2 above)
Comment (svalqui) [reply Bio/phenotype/pm_fitting.py]: Yes, either way they are hard to read.

**Comment by peterjc** [comment]
As an aside, a comment on your commit comments:

For the first commit, "Apply black style to Bio.phenotype." is fine.

For the second commit, "Apply black style to Bio.phenotype. Merge string." is misleading - all this commit did was merging strings.

I'll do a squash commit where "Apply black style to Bio.phenotype. Merge string." will make sense for a single combined commit.

**Comment by peterjc** [comment]
Merged, thank you.

**Pull request merged**